core/src/consensus/tendermint/mod.rs

```diff
@@ -74,12 +77,13 @@ pub struct Tendermint {
     lock_change: RwLock<Option<ConsensusMessage>>,
     /// Last lock view.
     last_lock: AtomicUsize,
-    /// Bare hash of the proposed block, used for seal submission.
```

Comment:
What's the difference between bare hash and hash?

Reply:
Bare hash does not contain seal.
This was used when a proposal block's seal is different from the imported block to get the same hash by not including the seal in the hash.
Now we use the same block as a proposal block and an imported block we do not need the bare hash.